src/user.rs

```diff
@@ -0,0 +1,177 @@
+//! Tasks defined rust_actix-basic.pdf
+//! Using endpoints at: https://jsonplaceholder.typicode.com/<API>
+
+use validator::Validate;
+use validator_derive;
+
+use reqwest;
+use serde::{Deserialize, Serialize};
+
+/// Submits a GET request to a specified URL, and returns a vector of users. Expected response is in json format, and can be parsed by the `User` struct with accessible elements by key name.
```

Comment:
Can you reformat, restricting your lines to a max of 80 characters.

Reply:
some comments have URLs that are above 80 chars... Otherwise, yes